utils/hct/hctbuild.cmd

```diff
@@ -213,6 +213,9 @@ if "%DO_SETUP%"=="1" (
   cmake %CMAKE_OPTS% -G %4 %HLSL_SRC_DIR% >> %3\cmake-log.txt 2>&1
   if errorlevel 1 (
     echo Failed to configure cmake projects.
+    echo ===== begin cmake-log.txt =====
+    type %3\cmake-log.txt
```

Comment:
This is general enough and also useful for building original DXC targets. So I think there is no need to wrap around with `#SPIRV change`, which should be used for SPIR-V specific modifications.